<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewdata</class>
 <widget class="QDialog" name="viewdata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查看历史收支</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#viewdata{background-image: url(:/new/myimage/new_icon/Background.jpg);}</string>
  </property>
  <widget class="QGroupBox" name="select_date_groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>411</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>选择日期</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>STKaiti</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>日期：</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="view_dateEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="OK_pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>STKaiti</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>STKaiti</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="view_listWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>210</y>
     <width>431</width>
     <height>331</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>431</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>431</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(187, 203, 216,120);</string>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="resizeMode">
    <enum>QListView::Adjust</enum>
   </property>
   <property name="batchSize">
    <number>120</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
